server/sonar-server/src/main/resources/org/sonar/server/rule/ws/example-search.json

```diff
@@ -7,6 +7,7 @@
       "key": "squid:S1067",
       "repo": "squid",
       "name": "Expressions should not be too complex",
+      "createdAt": "2013-03-27T08:52:40+0100",
```

Comment:
There have been "createdAt" fields on _some_ rules. I think it is better to have it on all or none rule.